code/game/machinery/PDApainter.dm

```diff
@@ -43,38 +43,25 @@
 		"Warden" = "pda-warden",
 		"Exploration Crew" = "pda-exploration"
 		)
-	var/id_icons = list(
-		"Assistant" = "id",
-		"Atmospheric Technician" = "atmos",
-		"Cargo" = "cargo",
-		"Chaplain" = "chap",
-		"Captain" = "captain",
-		"Chief Engineer" = "ce",
-		"Chief Medical Officer" = "cmo",
-		"Clown" = "clown",
-		"Detective" = "detective",
-		"Engineering" = "engi",
-		"Head of Personnel" = "hop",
-		"Head of Security" = "hos",
-		"Lawyer" = "lawyer",
-		"Medical" = "med",
-		"Mime" = "mime",
-		"Quartermaster" = "qm",
-		"Research Director" = "rd",
-		"Science" = "sci",
-		"Security" = "sec",
-		"Service" = "serv",
-		"Shaft Miner" = "miner",
-		"Warden" = "warden",
-		"Paramedic" = "paramed",
-		"Virologist" = "viro",
-		"Chemist" = "chemist",
-		"Geneticist" = "gene",
-		"Brig Physician" = "brigphys",
-		"Deputy" = "deputy",
-		"Roboticist" = "roboticist",
-		"Janitor" = "janitor",
-		"Exploration Crew" = "exploration"
+
+	var/valid_jobs = list(
+		// Command
+		"----Command----","Command (Custom)", "Captain", "Acting Captain",
```

Comment:
Won't this show "----Command----" on the list of options in the PDA painter? What happens if I try to pick it?

Reply:
Does nothing. It's boundaries for the selection window.